### PR TITLE
Create a migrate tips function

### PR DESCRIPTION
**Pull request by rohitpaulk**
For #3399, includes #3420

### CODE REVIEW

**Comment by whit537** [comment]
!m @rohitpaulk

**Comment by whit537** [comment]
Once #3415 settles down let's figure out how to merge https://github.com/gratipay/gratipay.com/issues/3399#issuecomment-102175471 and this.

**Comment by whit537** [comment]
Rebased on master.

File: gratipay/models/team.py [hunk 1]
```diff
@@ -54,3 +54,29 @@ def create_new(cls, owner, fields):
...
+        # Make sure the migration hasn't been done already
+        if subscriptions:
+            raise AlreadyMigrated
+
+        self.db.run("""
+
+            INSERT INTO subscriptions
+                        (ctime, mtime, subscriber, team, amount, is_funded)
+                 SELECT ctime
+                      , mtime
```
Comment (whit537) [inline gratipay/models/team.py]: I would leave this out and let `mtime` be set to `now`. No?

**Comment by whit537** [comment]
I suppose we want to run this when we approve a team for the first time.

**Comment by whit537** [comment]
How does this compare with the SQL I used the other day?

https://github.com/gratipay/gratipay.com/issues/3399#issuecomment-102175471

File: gratipay/models/team.py (hunk 1 above)
Comment (rohitpaulk) [inline gratipay/models/team.py]: Hmm, I'm not sure. We use `mtime` to show the time the tip was last modified in the UI (Last set). Do we want that to be the time when the user last set it, or the time when we migrated the tip?

**Comment by rohitpaulk** [comment]
> How does this compare with the SQL I used the other day?

That excludes tips from suspicious/closed/unclaimed accounts and also checks for `amount > 0`. Makes sense to exclude those here too.

**Comment by rohitpaulk** [comment]
> I suppose we want to run this when we approve a team for the first time.

Should we also send the tippers (`subscribers`, now) a notification?

**Comment by whit537** [comment]
@rohitpaulk Yes, definitely. Good call.

**Comment by whit537** [comment]
We should go back and notify the new subscribers from last week as well.

**Comment by rohitpaulk** [comment]
Rebased on master.

**Comment by rohitpaulk** [comment]
@whit537 - I'm going to use this for https://github.com/gratipay/gratipay.com/issues/3486#issuecomment-106622666, can you have a look and merge it in?

**Comment by whit537** [comment]
Yes, just reread it, looks good. Go ahead and merge once Travis runs green.

**Comment by rohitpaulk** [comment]
Deploying...

**Comment by rohitpaulk** [comment]
Deployed.

**Comment by whit537** [comment]
Awesome. Now to call it for the users in question?

**Comment by rohitpaulk** [comment]
I tested this on a backup, I can confirm that tips are migrated and an error is raised if the migration is attempted twice.

**Comment by rohitpaulk** [comment]
> Now to call it for the users in question?

Yep. Heading back to #3486...

**Pull request merged**